### PR TITLE
MON-19102-new

### PR DESCRIPTION
**Pull request by Noha-ElAbrouki**
**Description**: Replace “Sticky” by “Sticky for any non-OK status”.

**video**

[screen-capture (17).webm](https://github.com/user-attachments/assets/b4cbe94c-6f0e-4dd4-8693-d8c13f07c306)

### CODE REVIEW

**Review by Thebarda — DISMISSED** [review]

File: centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -17477,3 +17477,6 @@ msgstr "Erweiterter Modus"
 
 #  msgid "Poller ID #%d is the only one linked to poller/agent configuration ID #%d"
 #  msgstr ""
+
+# msgid "Sticky for any non-OK status"
+# msgstr ""
```
[the remark this answers is not in the document]
Comment (Noha-ElAbrouki) [reply centreon/lang/de_DE.UTF-8/LC_MESSAGES/messages.po:17482]: skip verification of empty translations for all languages  except  english and french

File: centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -18267,3 +18267,6 @@ msgstr "Você não salvou as alterações, deseja continuar?"
 
 #  msgid "Poller ID #%d is the only one linked to poller/agent configuration ID #%d"
 #  msgstr ""
+
+# msgid "Sticky for any non-OK status"
+# msgstr ""
```
[the remark this answers is not in the document]
Comment (Noha-ElAbrouki) [reply centreon/lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:18272]: skip verification of empty translations for all languages except english and french

File: centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -18281,3 +18281,6 @@ msgstr ""
 
 #  msgid "Poller ID #%d is the only one linked to poller/agent configuration ID #%d"
 #  msgstr ""
+
+# msgid "Sticky for any non-OK status"
+# msgstr ""
```
[the remark this answers is not in the document]
Comment (Noha-ElAbrouki) [reply centreon/lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:18286]: skip verification of empty translations for all languages except english and french

File: centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -17829,3 +17829,6 @@ msgstr "Retractar"
 
 #  msgid "Poller ID #%d is the only one linked to poller/agent configuration ID #%d"
 #  msgstr ""
+
+msgid "Sticky for any non-OK status"
+msgstr "Sticky para cualquier estado no-OK"
```
Comment (cg-tw) [inline centreon/lang/es_ES.UTF-8/LC_MESSAGES/messages.po:17834]:
```suggestion
msgstr "Sticky para cualquier estado que no sea OK"
```

**Review by cg-tw — DISMISSED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by quborg — APPROVED** [review]

**Review by mushroomempires — APPROVED** [review]

**Pull request merged**